tests/gocase/unit/type/bloom/bloom_test.go

```diff
@@ -116,4 +116,44 @@ func TestBloom(t *testing.T) {
 		}
 		require.LessOrEqual(t, float64(exist), fpp*float64(totalCount))
 	})
+
```

Comment:
Apart from this, we can use "INFO" to test the reserve and add

Reply:
Sure, there are some bugs about add.